UPGRADING

```diff
@@ -580,8 +580,10 @@ PHP 8.4 UPGRADE NOTES
     decimal number and round it to 0.29. However, "pre-rounding" incorrectly rounds
     certain numbers, so this fix removes "pre-rounding" and changes the way numbers
     are compared, so that the values are correctly rounded as decimal numbers.
-  . The maximum precision that can be handled by round() has been extended by
-    one digit.
+  . The maximum precision that can be handled by round() has been extended by one
+    digit. For example, if rounded 4503599627370495.5 with precision -1, would expect
+    it to result in 4503599627370496, but previously this was not the case and no
+    rounding was done.
```

Comment:
Maybe keep that more terse (and I don't think that the precision I mentioned is actually relevant here):
```suggestion
    digit. For example, `round(4503599627370495.5)` returned in `4503599627370495.5`,
    but now returns `4503599627370496`.
```